validator/val_utils.c

```diff
@@ -597,12 +619,23 @@ val_verify_new_DNSKEYs(struct regional* region, struct module_env* env,
 		BOGUS_KEY_TTL, *env->now);
 }
 
-enum sec_status 
+enum sec_status
 val_verify_DNSKEY_with_TA(struct module_env* env, struct val_env* ve,
 	struct ub_packed_rrset_key* dnskey_rrset,
 	struct ub_packed_rrset_key* ta_ds,
 	struct ub_packed_rrset_key* ta_dnskey, uint8_t* sigalg, char** reason,
 	struct module_qstate* qstate)
+{
+	return val_verify_DNSKEY_with_TA_ede(env, ve, dnskey_rrset, ta_ds,
+		ta_dnskey, sigalg, reason, NULL, qstate);
+}
+
+enum sec_status
+val_verify_DNSKEY_with_TA_ede(struct module_env* env, struct val_env* ve,
+	struct ub_packed_rrset_key* dnskey_rrset,
+	struct ub_packed_rrset_key* ta_ds,
+	struct ub_packed_rrset_key* ta_dnskey, uint8_t* sigalg, char** reason,
+	sldns_ede_code *reason_bogus, struct module_qstate* qstate)
```

Comment:
I would say that this function is not needed. We can just use `val_verify_DNSKEY_with_TA` and pass NULL in the places where *reason_bogus is not relevant (there are only a few).